README.md

```diff
@@ -1,8 +1,11 @@
 # Android Boilerplate
+<img src="https://raw.githubusercontent.com/SachseInTheHub/android-boilerplate/develop/android-ustwo.png" width="400">
```

Comment:
@Numan1617 you're a ninja. We were literally talking about this when the notification popped.